### DISCUSSION TITLE
Not able to select camera

### DISCUSSION THREAD

**Discussion by Mav977** [mebjas/html5-qrcode#495] [Q&A]
I have used the 'Easy mode' and according to screenshots and demos, I should get the option to select my camera. But I don't get that option not only on my laptop but also on my mobile. Moreover, on mobile, the camera uses the front camera which is not user-friendly to scan qr codes. Could you please help me out..

`<script src="https://unpkg.com/html5-qrcode" type="text/javascript"></script>`
` <center><div id="reader" style="width:300px"></div></center>`
<img width="661" alt="image" src="https://user-images.githubusercontent.com/100441348/175758456-74f539d5-6878-4af0-9d59-cb6f7d14508f.png">

<img width="1075" alt="image" src="https://user-images.githubusercontent.com/100441348/175759160-c6e9a8b2-62c0-4716-bc0f-69ddcdf8c087.png">
This is what I am getting

**Comment by ROBERT-MCDOWELL** [comment]
what do you call "options"? a select box with the list of devices?
